Gaijinizer patch/Map037.txt

```diff
@@ -833,8 +855,10 @@ No need to worry, I created them all anyways.
 
 こっちもさんざん面目潰されてキレてるんだぁ。
 > CONTEXT: Map037/events/4/pages/1/311/Dialogue/enemy3
-Do it all. I'm sorry that this is a very funny
-thing.#MTLed
+Try it.
+
+This one's face has also been completely cut and
+crushed.#StrateTemporary
```

Comment:
I checked with a native speaker, and it appears she's saying something to the effect of.

"Give it a try.

I'm pretty pissed off too about being humiliated like that."

I'm assuming referring to the Barghest taking Kazuya's side.